CONTRIBUTING.md

```diff
@@ -1,8 +1,9 @@
 **Steps to contribute icons to grommet-icons**
 
 1) Fork the grommet-icons project and clone.
-2) Add your raw svg files in this [folder](https://github.com/grommet/grommet-icons/tree/master/public/img) (use the other raw svgs as a reference on how to structure your file content and naming conventions).
-3) Run `yarn generate-icons` to generate the react components from the raw svg files from your grommet-icons fork.
-4) Consider adding search metadata to this [file](https://github.com/grommet/grommet-icons/blob/master/src/js/metadata.js) to help finding your new icon.
-5) Run `yarn test-update` to recreate the test snapshot files.
-6) Share with the community by submitting a PR.
+2) When adding new Icons the SVGs should be exported in a stroke version. This should include a stroke color as well as a stroke-width 
```

Comment:
```suggestion
2) When adding new icons,  the SVGs should be exported using stroke as opposed to fill. The SVG path should include `stroke="#000"` and `stroke-width="2"`.
```

Reply:
@taysea @britt6612 this looks good, would say "outline" instead of "fill". How about:

When adding new icons, export SVGs as stroke versions as opposed to outline. The SVG path should include `stroke="#000"` and `stroke-width="2"`.

Reply:
Thanks @bahriaditi I changed the following